upgrade/helpers/rhevm.py

```diff
@@ -262,14 +262,14 @@ def validate_and_create_product_templates():
     if sat_instance and cap_instance:
         sat_host = os.environ.get('RHEV_SAT_HOST')
         new_sat_template = os.environ.get('RHEV_SAT_IMAGE') + "_new"
-        cap_host = os.environ.get('RHEV_CAP_HOST')
+        cap_hosts = [os.environ.get('RHEV_CAP_HOST')]
         new_cap_template = os.environ.get('RHEV_CAP_IMAGE') + "_new"
         if check_necessary_env_variables_for_upgrade('capsule'):
             execute(check_ntpd, host=sat_host)
             execute(katello_restart, host=sat_host)
-            execute(check_capsule, cap_host, host=sat_host)
-            execute(check_ntpd, host=cap_host)
-            execute(katello_restart, host=cap_host)
+            execute(capsule_sync, cap_hosts, host=sat_host)
+            execute(check_ntpd, host=cap_hosts[0])
```

Comment:
Why are you checking it on only the first capsule and not others, you should check them on all. Get this in loop

Reply:
These tasks won't work for user specified boxes , as we won't know the user image names , env , storage domain , in short , their RHEV env

Reply:
do it for single capsule.